### PR TITLE
Cross-reference multi-zone and single-zone documentation

### PR DESCRIPTION
**Pull request by rosbo**
Also fix the API link for `google_compute_instance_group_manager` documentation.

cc/ @selmanj

### CODE REVIEW

File: website/docs/r/compute_region_instance_group_manager.html.markdown [hunk 1]
```diff
@@ -13,6 +13,8 @@ of homogeneous Compute Engine virtual machine instances from a common instance
 template. For more information, see [the official documentation](https://cloud.google.com/compute/docs/instance-groups/distributing-instances-with-regional-instance-groups)
 and [API](https://cloud.google.com/compute/docs/reference/latest/regionInstanceGroupManagers)
 
+~> **Note:** Use [google_compute_instance_group_manager](/docs/providers/google/r/google_compute_instance_group_manager.html) to create a single-zone instance group manager.
```
Comment (selmanj) [inline website/docs/r/compute_region_instance_group_manager.html.markdown]: Is '~>' valid markdown? I've never seen that before

Comment (selmanj) [reply website/docs/r/compute_region_instance_group_manager.html.markdown]: If it's not I'd prefer to drop it as the ** emphasis should be enough

Comment (rosbo) [reply website/docs/r/compute_region_instance_group_manager.html.markdown]: It is an extension of markdown. It creates a block with a light yellow background. You can see an example here:
https://www.terraform.io/docs/providers/google/r/compute_network_peering.html

We uses it already in 9 different pages in our docs.

Comment (selmanj) [reply website/docs/r/compute_region_instance_group_manager.html.markdown]: Thanks, it's completely new to me.

**Pull request merged**